### PR TITLE
New/check if user is logged in

### PR DESCRIPTION
**Pull request by Burbinox**
Check if user is logged in (for now check only /logout endpoint)

### CODE REVIEW

File: server/controllers/user.js [hunk 1]
```diff
@@ -19,31 +20,32 @@ export default class UserController {
...
+      const sessionToken = crypto.randomBytes(64).toString("base64");
       await client.query(`UPDATE users SET session=$1 WHERE username=$2;`, [
-        JSON.stringify(session),
-        body.username,
+        sessionToken,
+        req.body.username,
       ]);
+      res.cookie("team-lp-project-5", sessionToken);
     }
-    return { message: "Logged in successfully." };
```
Comment (patrykbrodziak1) [inline server/controllers/user.js:46]: It should be visible

Comment (Burbinox) [reply server/controllers/user.js:46]: Fixed

**Comment by patrykbrodziak1** [comment]
Looks good for me

**Review by patrykbrodziak1 — APPROVED** [review]

**Review by pastepi — APPROVED** [review]
I think all is 🆗

**Pull request merged**